swap_meet/clothing.py

```diff
@@ -1,2 +1,31 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
```

Comment:
Nice work importing `Item` since we're using by name as the parent class of Clothing 👍